chef/cookbooks/monasca/templates/default/log-agent.conf.erb

```diff
@@ -0,0 +1,38 @@
+input {
+  # TODO(trebskit) should initial list contain some files to be
```

Comment:
Sensible defaults are good, yes. Also, these need to be retrieved from a barclamp attribute so they can be adjusted by the users according to their needs (right now they are static).